server/utils/dateUtils.ts

```diff
@@ -50,6 +50,14 @@ export class DateFormats {
     return formatISO(date)
   }
 
+  /**
+   * @param date JS Date object.
+   * @returns the time in the format '19:05'.
+   */
+  static dateObjToIsoTime(date: Date) {
```

Comment:
Pedantic nitpick of the day: `19:05` isn't actually ISO time -- that would be `19:05:00(.000)Z`, while `19:05` is 24hr time 😄